From 4cbd91df76171f06e8549ead0898e60abd70eb7b Mon Sep 17 00:00:00 2001
From: Varun Sharma <vsharma@habana.ai>
Date: Wed, 20 May 2020 22:40:25 +0300
Subject: [PATCH] [SW-0] goya2: init.d: disable i2c based init

In current goya2 onix i2c is disabled, which
causes errors while executing init script during
linux boot, as its tries to use i2cget/i2cset,
for which entries are not available.
Currently disabling for all goya2 board types.

Change-Id: I4d03f347cd22d72fb485548ab5597283ac19a676
Signed-off-by: Varun Sharma <vsharma@habana.ai>
---

diff --git a/board/habanalabs/common/rootfs_overlay/etc/init.d/S80i2c-instantiate b/board/habanalabs/common/rootfs_overlay/etc/init.d/S80i2c-instantiate
index 973c79d..15579ed 100755
--- a/board/habanalabs/common/rootfs_overlay/etc/init.d/S80i2c-instantiate
+++ b/board/habanalabs/common/rootfs_overlay/etc/init.d/S80i2c-instantiate
@@ -36,6 +36,9 @@
 	local prod_num=0x`grep "System Product HL=" $eeprom_data | cut -d= -f2`
 
 	case "$device" in
+	goya2)
+	return 0
+	;;
 	goya)
 		if [ $hl_ver -ge 4 ]; then
 			echo tmp411 0x4d > /sys/bus/i2c/devices/i2c-1/new_device
